src/steps/issues.ts

```diff
@@ -65,81 +65,86 @@ export async function fetchIssues({
 
-        if (issue.fields.creator && issue.fields.creator.accountId) {
-          const creatorUserKey = generateEntityKey(
-            USER_ENTITY_TYPE,
-            issue.fields.creator.accountId,
+          const projectEntity = projectEntities?.find(
+            (project) => project.key === projectConfig.key,
           );
-          const creatorEntity = (await jobState.findEntity(
-            creatorUserKey,
-          )) as UserEntity;
-
-          if (creatorEntity) {
+          if (projectEntity) {
             await jobState.addRelationship(
               createDirectRelationship({
-                _class: RelationshipClass.CREATED,
-                from: creatorEntity,
+                _class: RelationshipClass.HAS,
+                from: projectEntity,
                 to: issueEntity,
               }),
             );
           } else {
             logger.warn(
-              { creatorUserKey, issueName: issueEntity.name },
-              'Created user is no longer in this Jira instance. Not creating user_created_issue relationship.',
+              { projectKey: projectConfig.key },
+              'Unable to create issue -> project relationship because the project was not in the job state',
             );
           }
-        }
-
-        if (issue.fields.reporter && issue.fields.reporter.accountId) {
-          const reporterUserKey = generateEntityKey(
-            USER_ENTITY_TYPE,
-            issue.fields.reporter.accountId,
-          );
-          const reporterEntity = (await jobState.findEntity(
-            reporterUserKey,
-          )) as UserEntity;
 
-          if (reporterEntity) {
-            await jobState.addRelationship(
-              createDirectRelationship({
-                _class: RelationshipClass.REPORTED,
-                from: reporterEntity,
-                to: issueEntity,
-              }),
+          if (issue.fields.creator && issue.fields.creator.accountId) {
+            const creatorUserKey = generateEntityKey(
+              USER_ENTITY_TYPE,
+              issue.fields.creator.accountId,
             );
-          } else {
-            logger.warn(
-              { reporterUserKey, issueName: issueEntity.name },
-              'Reported user is no longer in this Jira instance. Not creating user_reported_issue relationship.',
+            const creatorEntity = (await jobState.findEntity(
+              creatorUserKey,
+            )) as UserEntity;
+
+            if (creatorEntity) {
+              await jobState.addRelationship(
+                createDirectRelationship({
+                  _class: RelationshipClass.CREATED,
+                  from: creatorEntity,
+                  to: issueEntity,
+                }),
+              );
+            } else {
+              logger.warn(
+                { creatorUserKey, issueName: issueEntity.name },
+                'Created user is no longer in this Jira instance. Not creating user_created_issue relationship.',
+              );
```

Comment:
We should avoid `logger.warn` unless we intend to monitor and do something about these log events.
```suggestion
              logger.info(
                { creatorUserKey, issueName: issueEntity.name },
                '[SKIP] user_created_issue relationship, issue creator not found',
              );
```
Similar changes are necessary for other parts of this file.

Reply:
Gotcha. I like this update. Thanks. Now that I've watched you and Austin use Cloudwatch more, these kinds of improvements are making more sense to me.